<?xml version="1.0" ?>
<testsuite errors="0" failures="0" name="django.contrib.auth.tests.auth_backends.InActiveUserBackendTest" tests="2" time="0.002">
	<testcase classname="django.contrib.auth.tests.auth_backends.InActiveUserBackendTest" name="test_has_module_perms" time="0.001"/>
	<testcase classname="django.contrib.auth.tests.auth_backends.InActiveUserBackendTest" name="test_has_perm" time="0.001"/>
	<system-out/>
	<system-err/>
</testsuite>
